code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -36,20 +36,25 @@
 	flags_pass = PASSTABLE
 	use_plasma(10)
 	throw_at(T, 6, 2, src) //Victim, distance, speed
-	spawn(6)
-		if(!xeno_caste.hardcore)
-			flags_pass = initial(flags_pass) //Reset the passtable.
-		else
-			flags_pass = 0 //Reset the passtable.
-
-	spawn(xeno_caste.pounce_delay)
-		usedPounce = 0
-		to_chat(src, "<span class='notice'>You get ready to pounce again.</span>")
-		update_action_button_icons()
+	addtimer(CALLBACK(src, /mob/living/carbon/Xenomorph/proc/reset_flags_pass), 6)
+	addtimer(CALLBACK(src, /mob/living/carbon/Xenomorph/proc/reset_pounce_delay), xeno_caste.pounce_delay)
 
 	return TRUE
 
+/mob/living/carbon/Xenomorph/proc/reset_pounce_delay()
+	usedPounce = FALSE
+	to_chat(src, "<span class='notice'>You get ready to pounce again.</span>")
+	update_action_button_icons()
 
+/mob/living/carbon/Xenomorph/Hunter/reset_pounce_delay()
+	. = ..()
+	playsound(src, 'sound/effects/xeno_newlarva.ogg', 50, 0, 1)
+
+/mob/living/carbon/Xenomorph/proc/reset_flags_pass()
+	if(!xeno_caste.hardcore)
+		flags_pass = initial(flags_pass) //Reset the passtable.
+	else
+		flags_pass = 0 //Reset the passtable.
```

Comment:
Don't we have a NONE define to show its a bit field? If so we might wanna add one for clarity